workflow/rest/runs/logs.mdx

```diff
@@ -0,0 +1,340 @@
+---
+title: "List events"
+description: "Fetch flat event logs of workflow runs"
+api: "GET https://qstash.upstash.io/v2/workflows/events"
+authMethod: "bearer"
+---
+
+<Warning>
+  The flat events provide a more detailed version of the [List Workflow Events API](/workflow/rest/runs/event-logs). If you need an overview of a workflow run, that API is more useful, as it returns grouped and processed data.
+
+  This flat view is primarily useful for retrieving error logs and more detailed step logs that are not available in the compact workflow run information.
+</Warning>
+
+Each step in a workflow is associated with a message. During its lifecycle, a message generates multiple logs.
+
+For example, when you trigger a workflow, an initial message is created to execute the first step. This message logs a `CREATED` state and then waits to be processed by the server. Once successfully processed, it logs a `DELIVERED` state. Some data may be duplicated across these events.
+
+In addition to logs generated by messages, we also log events at the workflow run level. For instance, when a workflow is triggered, we log a `RUN_STATED` state. Similarly, we log `RUN_FAILED`, `RUN_SUCCESS`, and other states to track execution status.
+
+The flat view returns all logs generated by a workflow run in their raw format. By checking the flat view, you can see all the lifecycle of workflow run.
+
+This means that if you fetch logs with the `RUN_SUCCESS` state, you will get a list of completed workflow runs.
+Similarly, if you fetch logs with the `RUN_STARTED` state, you will see a list of workflow runs that were triggered. However, this does not necessarily mean they are still in progress, as both of these logs are generated at specific points in the workflow lifecycle.
+
+## Request
+
+<ParamField query="cursor" type="string">
+  By providing a cursor you can paginate through all of the workflow runs.
+</ParamField>
+
+<ParamField query="workflowRunId" type="string">
+  Filters logs by workflow run ID.
+</ParamField>
+
+<ParamField query="workflowUrl" type="string">
+  Filters logs by the workflow server URL.
+</ParamField>
+
+<ParamField query="workflowCreatedAt" type="number">
+  Filters logs by creation date of workflow run (Unix timestamp in milliseconds).
+</ParamField>
+
+<ParamField query="state" type="string">
+  Filter logs by [state](/qstash/howto/debug-logs)
```

Comment:
These are the states of the Workflow's right. Not the messages. 
I guess we need to remove the link and mention the differnce.

Reply:
I think, I entered the comment on the wrong API. 
This API for flat events. That means, user can enter message events and also `RUN_STARTED` , `WAITER_ADDED` etc. So the linked doc is still not enough.